docs/examples/lynx/scale-feedback/preview.css

```
@@ -0,0 +1,52 @@
+/* biome-ignore lint/correctness/noUnknownTypeSelector: page is a Lynx element. */
+page {
```

Stylelint (17.14.0)

[error] Unknown type selector "page" (selector-type-no-unknown)

(selector-type-no-unknown)